packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1997,7 +2001,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             // Wrap data store summaries in .channels subtree.
             wrapSummaryInChannelsTree(summarizeResult);
         }
-        this.addContainerStateToSummary(summarizeResult);
+        this.addContainerStateToSummary(
+            summarizeResult,
+            true, // fullTree
+            false); // trackState
```

Comment:
nit: trackState should be next to false, i.e., inside the parenthesis. Other parts of the code also use /* trackState */ kind of comments.